packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -87,6 +87,55 @@ export default /* @ngInject */ ($stateProvider) => {
       terminateProject: /* @ngInject */ (OvhApiCloudProject) => (project) =>
         OvhApiCloudProject.v6().delete({ serviceName: project.serviceName })
           .$promise,
+
+      meApiSchemas: /* @ngInject */ (PciProjectsService) =>
+        PciProjectsService.getMeSchemas(),
+
+      buildMapEnum: /* @ngInject */ (meApiSchemas) => (path) => {
+        const enums = Object.entries(meApiSchemas.models[path].enum);
+        return Object.freeze(
+          enums.reduce(
+            (results, statusEnum) => ({
+              [statusEnum[1]]: statusEnum[1],
+              ...results,
+            }),
+            {},
+          ),
+        );
+      },
```

Comment:
@cbourgois , changes are coming from here:
https://github.com/ovh/manager/pull/5114#discussion_r652414276

Reply:
Also, yes sure it will break if API change the schema.
But it will the break even if  we have them into const from our side, because we will compare our const value to something that has been changed or removed from API side.

Reply:
IMO, You can use enum from API if you want to have a list based on enums (`<select` or `ng-repeat`) and I think it's the purpose of the comment you've linked.
If you want to compare a status with an enum value, there is no other solution than comparing to a string.
Here you build an enum object dynamically based on API schema, and just after you are using an explicit enum value. It will break and make no sense.
And IMHO the code is not easy to understand.